src/components/Table/GridTable.tsx

```diff
@@ -483,7 +483,9 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   return (
     <RowStateContext.Provider value={rowStateContext}>
       <PresentationProvider fieldProps={fieldProps} wrap={style?.presentationSettings?.wrap}>
-        <div ref={resizeRef} css={Css.w100.$} />
+        {/* If virtualized take some pixels off the width to accommodate when virtuoso's scrollbar is introduced. */}
+        {/* Otherwise a horizontal scrollbar will _always_ appear once the vertical scrollbar is needed */}
+        <div ref={resizeRef} css={Css.w100.if(as === "virtual").w("calc(100% - 20px)").$} />
```

Comment:
I had noticed that a lot of tables in Blueprint are adding a horizontal scrollbar for very little content. The reason is that this `resizeRef` which we use to determine to measure the available space is outside of the Virtuoso scroller. So when Virtuoso adds in a scrollbar, the size of the table isn't adjusted and instead it requires the horizontal scrollbar to see all of the content. 
Writing this out makes me wonder if we should instead conditionally use `resizeRef` or the virtuoso scroller ref to do these calculations from.. Probably that would be a better way of going instead of just saying "remove 20px, maybe? that sounds good"